### DISCUSSION TITLE
Main menu change from "Print" to "Media" ?

### DISCUSSION THREAD

**Discussion by GT-Geo** [mriscoc/Ender3V2S1#421] [Q&A] [answered]
Why was the main menu changed from "Print" to "Media"?  I much prefer the way it was originally.

**Comment by ghost** [comment, marked as answer]
I believe it's because this menu only serves to use the sdcard

**Reply by mriscoc** [reply]
Yes, and also because it can be used to load configuration files, that is, not just printable files.

**Reply by GT-Geo** [reply]
That makes sense now that it can be used to load config files. Thanks
